cpp/src/gandiva/gdv_function_stubs.h

```diff
@@ -55,119 +55,119 @@ using gdv_month_interval = int32_t;
                                int64_t* dec_high_from_str, uint64_t* dec_low_from_str);
 
-char* gdv_fn_dec_to_string(int64_t context, int64_t x_high, uint64_t x_low,
+char* gdv_fn_dec_to_string(void* context_ptr, int64_t x_high, uint64_t x_low,
                            int32_t x_scale, int32_t* dec_str_len);
 
 GANDIVA_EXPORT
-int32_t gdv_fn_castINT_utf8(int64_t context, const char* data, int32_t data_len);
+int32_t gdv_fn_castINT_utf8(void* context_ptr, const char* data, int32_t data_len);
 
 GANDIVA_EXPORT
-int64_t gdv_fn_castBIGINT_utf8(int64_t context, const char* data, int32_t data_len);
+int64_t gdv_fn_castBIGINT_utf8(void* context_ptr, const char* data, int32_t data_len);
 
 GANDIVA_EXPORT
-float gdv_fn_castFLOAT4_utf8(int64_t context, const char* data, int32_t data_len);
+float gdv_fn_castFLOAT4_utf8(void* context_ptr, const char* data, int32_t data_len);
 
 GANDIVA_EXPORT
-double gdv_fn_castFLOAT8_utf8(int64_t context, const char* data, int32_t data_len);
+double gdv_fn_castFLOAT8_utf8(void* context_ptr, const char* data, int32_t data_len);
 
 GANDIVA_EXPORT
-const char* gdv_fn_castVARCHAR_int32_int64(int64_t context, int32_t value, int64_t len,
+const char* gdv_fn_castVARCHAR_int32_int64(void* context_ptr, int32_t value, int64_t len,
                                            int32_t* out_len);
 GANDIVA_EXPORT
-const char* gdv_fn_castVARCHAR_int64_int64(int64_t context, int64_t value, int64_t len,
+const char* gdv_fn_castVARCHAR_int64_int64(void* context_ptr, int64_t value, int64_t len,
                                            int32_t* out_len);
 GANDIVA_EXPORT
-const char* gdv_fn_castVARCHAR_float32_int64(int64_t context, float value, int64_t len,
+const char* gdv_fn_castVARCHAR_float32_int64(void* context_ptr, float value, int64_t len,
                                              int32_t* out_len);
 GANDIVA_EXPORT
-const char* gdv_fn_castVARCHAR_float64_int64(int64_t context, double value, int64_t len,
+const char* gdv_fn_castVARCHAR_float64_int64(void* context_ptr, double value, int64_t len,
                                              int32_t* out_len);
 
 GANDIVA_EXPORT
 int32_t gdv_fn_utf8_char_length(char c);
 
 GANDIVA_EXPORT
-const char* gdv_fn_upper_utf8(int64_t context, const char* data, int32_t data_len,
+const char* gdv_fn_upper_utf8(void* context_ptr, const char* data, int32_t data_len,
                               int32_t* out_len);
 
 GANDIVA_EXPORT
-const char* gdv_fn_lower_utf8(int64_t context, const char* data, int32_t data_len,
+const char* gdv_fn_lower_utf8(void* context_ptr, const char* data, int32_t data_len,
                               int32_t* out_len);
 
 GANDIVA_EXPORT
-const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_len,
+const char* gdv_fn_initcap_utf8(void* context_ptr, const char* data, int32_t data_len,
                                 int32_t* out_len);
 
 GANDIVA_EXPORT
-int32_t gdv_fn_castINT_varbinary(gdv_int64 context, const char* in, int32_t in_len);
```

Comment:
why was this gdv_int64 instead of int64_t earlier? guessing they are of the same type.